mobile/lib/ui/viewer/actions/file_selection_actions_widget.dart

```diff
@@ -602,6 +607,44 @@ class _FileSelectionActionsWidgetState
     }
   }
 
+  Future<String> saveImage(Uint8List bytes) async {
+    String path = "";
+    try {
+      final Directory root = await getTemporaryDirectory();
+      final String directoryPath = '${root.path}/enteTempFiles';
+      // Create the directory if it doesn't exist
+      final DateTime timeStamp = DateTime.now();
+      await Directory(directoryPath).create(recursive: true);
+      final String filePath = '$directoryPath/$timeStamp.jpg';
+      final file = await File(filePath).writeAsBytes(bytes);
```

Comment:
Can you check if this file is automatically getting deleted? I'm not sure what happens. If not, we have to delete it manually after use.